### DISCUSSION TITLE
v3.0.0-rc.1

### DISCUSSION THREAD

**Discussion by pi0** [nuxt/nuxt#18385] [Announcements]
#  🐣  `3.0.0-rc.1` ["Mount Hope"](https://en.wikipedia.org/wiki/Mount_Hope_(Palmer_Land))

## Release Announcement

🚀 Check out the [Announcing Nuxt 3 Release Candidate](https://nuxtjs.org/announcements/nuxt3-rc) for more information about this release!

## Migrating from `nuxt3` edge releases

If you are already a beta tester of nuxt using `nuxt3` package, please follow the steps below to use RC.1:

1. Check out [Breaking changes tracker](https://github.com/nuxt/framework/discussions/2883) to apply any of the previous possible breaking changes on edge-channel.

1. Remove `.nuxt`, `package.lock.json`, `yarn.lock` and `node_modules`

1. Update `nuxt.config` to import `defineNuxtConfig` from `nuxt` instead of `nuxt3`:

```diff
- import { defineNuxtConfig } from 'nuxt3'
+ import { defineNuxtConfig } from 'nuxt'

export default defineNuxtConfig({ ... })
```

1. Update `package.json`

```diff
{
  "devDependencies": {
-    "nuxt3": "latest"
+    "nuxt": "^3.0.0-rc.1"
  }
}
```

Alternatively, you can use `"nuxt": "npm:nuxt3@latest"` to keep using the latest edge releases of Nuxt 3.

## Migration for module authors

For Nuxt 3 modules created by new [module-builder](https://github.com/nuxt/module-builder) toolchain, also update `@nuxt/kit` dependency to use RC.1 in `package.json`:

```diff
{
  "dependencies": {
-    "@nuxt/kit": "npm:@nuxt/kit-edge@latest"
+    "@nuxt/kit": "^3.0.0-rc.1"
  }
}
```

<hr /><em>This discussion was created from the release <a href='https://github.com/nuxt/framework/releases/tag/v3.0.0-rc.1'>v3.0.0-rc.1</a>.</em>

**Comment by mctweb** [comment]
Congratulation team Nuxt! Exciting times for developers, thank you!

**Comment by markgibbons25** [comment]
Awesome, easy update from edge on my project.

**Reply by miscab** [reply]
Will try tomorrow, wonderful! Million thanks to Team.

**Comment by maxdzin** [comment]
My congratulations and big love to the team!
Thank you for such amazing work! ❤️

**Comment by edsoncanganjo** [comment]
It’s amazing thank you.

**Comment by NyllRE** [comment]
Nuxt and it's team has been doing really great lately. Can't wait for LTS releases. ❤️

**Comment by thomasbnt** [comment]
Congrats, team Nuxt ! 🎉🎉🎉

**Comment by gxmari007** [comment]
Nice work!

**Comment by Slaknoah** [comment]
Congratulations to the team!  🎉🎉🎉

**Comment by sjonibek** [comment]
Awesome! Thanks to the Team!

**Comment by hooman-mirghasemi** [comment]
I wish nuxt js, like the quasar in the new version, moves towards giving outputs for Android, iOS, etc.
If being a cross-platform is added to nuxt features, It will become a big giant

**Reply by pi0** [reply]
Our main focus is towards a stable full-stack Vue3 app support currently but I don't see it an impossible goal that we can make this possible during 3.x releases for native app support since nitro output is pretty much portable.
